tests/components/homekit_controller/common.py

```diff
@@ -252,26 +253,21 @@ async def device_config_changed(hass: HomeAssistant, accessories: Accessories):
     accessories_obj = Accessories()
     for accessory in accessories:
         accessories_obj.add_accessory(accessory)
-    pairing._accessories_state = AccessoriesState(
-        accessories_obj, pairing.config_num + 1
-    )
+
+    new_config_num = pairing.config_num + 1
     pairing._async_description_update(
-        HomeKitService(
-            name="TestDevice.local",
+        AbstractDescription(
+            name="testdevice.local.",
             id="00:00:00:00:00:00",
-            model="",
-            config_num=2,
-            state_num=3,
-            feature_flags=0,
             status_flags=0,
+            config_num=new_config_num,
             category=1,
-            protocol_version="1.0",
-            type="_hap._tcp.local.",
-            address="127.0.0.1",
-            addresses=["127.0.0.1"],
-            port=8080,
```

Comment:
This was actually sending the wrong object, but it had the properties that were needed so it worked